src/SDK/SmartDetectorsShared/Clients/AnalysisServicesFactory.cs

```diff
@@ -17,6 +17,8 @@ namespace Microsoft.Azure.Monitoring.SmartDetectors.Clients
     using Microsoft.Azure.Monitoring.SmartDetectors.ActivityLog;
     using Microsoft.Azure.Monitoring.SmartDetectors.Arm;
     using Microsoft.Azure.Monitoring.SmartDetectors.Metric;
+    using Newtonsoft.Json;
```

Comment:
>using Newtonsoft.Json [](start = 4, length = 21)

better remove this file from the change - i'm about to change the whole thing to resource centric